### PR TITLE
Mosip-18244: Rid status service impl code coverage improved.

### PR DESCRIPTION
**Pull request by kameshsr**

### CODE REVIEW

**Review by m1054352 — APPROVED** [review]

File: resident/resident-service/src/test/java/io/mosip/resident/test/service/RidStatusServiceTest.java [hunk 1]
```diff
@@ -111,6 +111,17 @@ public void getRidStatusTest() throws ApisResourceAccessException {
 		responseWrapper.setResponse(list);
 		residentService.getRidStatus(requestDTO);
 
+		// UIN_GENERATED
+		response.setStatusCode("UIN_GENERATED");
+		responseWrapper.setResponse(list);
+		residentService.getRidStatus(requestDTO);
```
Comment (vishwa-vyom) [inline resident/resident-service/src/test/java/io/mosip/resident/test/service/RidStatusServiceTest.java]: Why this is no assert statements ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: resident/resident-service/src/test/java/io/mosip/resident/test/service/RidStatusServiceTest.java (hunk 1 above)
Comment (kameshsr) [reply resident/resident-service/src/test/java/io/mosip/resident/test/service/RidStatusServiceTest.java]: yes sir i have added now in this commit. https://github.com/mosip/resident-services/pull/227/commits/aedb823be1437fa4ed2061104b57eec75ad1ef4f

Comment (vishwa-vyom) [reply resident/resident-service/src/test/java/io/mosip/resident/test/service/RidStatusServiceTest.java]: Why residentService.getRidStatus(requestDTO) is called twice ?

Comment (kameshsr) [reply resident/resident-service/src/test/java/io/mosip/resident/test/service/RidStatusServiceTest.java]: Yes sir for each validation response like UIN_GENERATED and AWAITING_INFORMATION separate variable required to test different assert conditions.

**Review by m1054352 — APPROVED** [review]

**Pull request merged**